Rakefile

```diff
@@ -223,6 +232,25 @@ def platform
   end
 end
 
+def ios_build_config
+  signing_style = IOS_SIGNING_STYLE
+  team_id = ENV['IOS_DEVELOPMENT_TEAM_ID']
+
+  if(environment === 'production')
+    provisioning_profile = ENV['PROVISIONING_PROFILE_PROD']
```

Comment:
Just a note in case of Donor APP we can keep `PROVISIONING_PROFILE_PROD ` as is or we can follow the same convention of `PROVISIONING_PROFILE_DONOR_PROD `

Reply:
I think "PROVISIONING_PROFILE_PROD" is fine as we only have one provisioning profile.